versions/3.0.4.md

```diff
@@ -2302,7 +2303,7 @@ For computing links and providing instructions to execute them, a [runtime expre
 
 | Field Name | Type | Description |
 | ---- | :----: | ---- |
-| <a name="link-operation-ref"></a>operationRef | `string` | A relative or absolute URI reference to an OAS operation. This field is mutually exclusive of the `operationId` field, and MUST point to an [Operation Object](#operation-object). Relative `operationRef` values MAY be used to locate an existing [Operation Object](#operation-object) in the OpenAPI description. |
+| <a name="link-operation-ref"></a>operationRef | `string` | A relative or absolute URI reference to an OAS operation. This field is mutually exclusive of the `operationId` field, and MUST point to an [Operation Object](#operation-object). Relative `operationRef` values MAY be used to locate an existing [Operation Object](#operation-object) in the OpenAPI Description. |
```

Comment:
Does this mean that `operationRef` can only point to a resource/document that is _already part of the OAD due to the action of another keyword_?  This is relevant to the text early in the spec that states what keywords are allowed to connect parts into an OAD. (but see next comment in the Reference Object)

Reply:
I don't read it that way. Though I do find the terminology "A relative or absolute URI reference" a bit strange. I _think_ that URI references are either relative refs or URIs, and URIs may be absolute or not, but I don't think "an absolute URI reference" is a thing. This is the only place this phrase occurs in the spec. I think "URI reference" is clearer and more accurate, so I'll make that change. Also open to additional changes if you think they are needed.

Reply:
"URI reference" in RFC3986 is the most generic term.  Anything vaguely URI-ish (relative, absolute, full) is technically A URI reference (often written "URI-reference"). 

But you are right that this needs to be changed for a different reason:  `https://example.com/foo#/components/pathItems/bar/get` is a URI-reference that is neither relative (because it has a scheme) nor absolute (because it has a fragment).